### ISSUE TITLE
Frame interpolation causes screen jitter, and screen capture fails under Root mode.

### ISSUE THREAD

**Issue by nedalverta** [FrankBarretta/LSFG-Android#35]
The Dimensity 9400+ cannot capture screen content in Root mode or Shizuku mode. It works in Accessibility mode, but the screen exhibits severe flickering.Version: 0.1.1

**Comment by FrankBarretta** [comment]
With the new fix release, I should have solved the problem. See if you still have the problem.

**Comment by nedalverta** [comment]
> With the new fix release, I should have solved the problem. See if you still have the problem.

Thanks for your reply, developer. However, after my testing, the issue still hasn't been fixed. Attached please find the log below.

[lsfg.txt](https://github.com/user-attachments/files/27309644/lsfg.txt)

OnePlus ACE5 Ultimate Edition (Mali-G925 + Android 15)
